Library/Formula/flatcc.rb

```diff
@@ -0,0 +1,54 @@
+class Flatcc < Formula
+  desc "FlatBuffers Compiler and Library in C for C"
+  homepage "https://github.com/dvidelabs/flatcc"
+  url "https://github.com/dvidelabs/flatcc/archive/v0.2.0.tar.gz"
+  sha256 "5327e0c0faa37643f59be6f086576ef76a13d36b655d2bf9547f89ffce3b3413"
+  head "https://github.com/dvidelabs/flatcc.git"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", "-G", "Unix Makefiles", buildpath, *std_cmake_args
+    system "make"
+
+    bin.install "bin/flatcc"
```

Comment:
There's no `make install` target. No `distclean` either. The [README says](https://github.com/dvidelabs/flatcc/blob/e0d17df7dd16a38fe24bb941592da8b9128598a6/README.md#building) to use their custom build scripts in `scripts/`, which use the `ninja` build tool. But that doesn't work for me out of the box.

```
[~/tmp/flatcc/flatcc-0.2.0]
$ ./scripts/build.sh
building Debug
ninja: error: loading 'build.ninja': No such file or directory
```

This seems a little wonky. Might be worth an upstream bug report on it.